components/BottomNavigation/src/MDCBottomNavigationBar.m

```diff
@@ -76,6 +76,9 @@ @interface MDCBottomNavigationBar () <MDCInkTouchControllerDelegate,
 
 @implementation MDCBottomNavigationBar
 
+@synthesize mdc_overrideBaseElevation = _mdc_overrideBaseElevation;
```

Comment:
Isn't it defaulting to the underscore, can't we omit the `= _mdc...` part?

Reply:
If I don't specify the ivar name, it seemed to synthesize it to `mdc_overrideBaseElevation`. 

<img width="1133" alt="Screen Shot 2019-07-23 at 11 24 36 PM" src="https://user-images.githubusercontent.com/1753199/61762777-0e229c80-ada1-11e9-9dbb-2343d05a5bf4.png">

Reply:
sweet, thank you